geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/udf/SpatialRelationFunctions.scala

```diff
@@ -39,8 +39,7 @@ object SpatialRelationFunctions {
   val ST_Touches:    (Geometry, Geometry) => jl.Boolean = nullableUDF((geom1, geom2) => geom1.touches(geom2))
   val ST_Within:     (Geometry, Geometry) => jl.Boolean = nullableUDF((geom1, geom2) => geom1.within(geom2))
   val ST_Relate:     (Geometry, Geometry) => String = nullableUDF((geom1, geom2) => geom1.relate(geom2).toString)
-  val ST_RelateBool: (Geometry, Geometry, String) => Boolean =
-    nullableUDF((geom1, geom2, pattern) => geom1.relate(geom2, pattern))
+  val ST_RelateBool: (Geometry, Geometry, String) => jl.Boolean = nullableUDF((geom1, geom2, pattern) => geom1.relate(geom2, pattern))
```

Comment:
@elahrvivaz I slipped this change in there, which was the main impetus for [issue 2174](https://geomesa.atlassian.net/browse/GEOMESA-2174). This change makes it boxed, the opposite of what I was advocating for in the issue text. Still want to fine-tooth the code for any more.